examples/seismic/tutorials/16_ader_fd.ipynb

```diff
@@ -0,0 +1,565 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    
</pre><p> </p><p>I still don't quite get why using <code>grad(div)</code> and such doesn't work.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>